apps/examples/testcase/ta_tc/task_manager/itc/itc_task_manager_main.c

```diff
@@ -104,38 +110,72 @@ void unregister_tasks_4(int gid)
 	int ret;
 
 	group_list_info = (tm_appinfo_list_t *)task_manager_getinfo_with_group(gid, TM_RESPONSE_WAIT_INF);
-	TC_ASSERT_NEQ("task_manager_getinfo_with_group", group_list_info, NULL);
+	if (group_list_info == NULL) {
+		printf("ERROR: failed task_manager_getinfo_with_group in unregister_tasks_4\n");
+		return;
+	}
 
 	while (group_list_info != NULL) {
+
 		task_info = &(group_list_info->task);
+		ret = task_manager_stop(task_info->handle, TM_RESPONSE_WAIT_INF);
+		if (ret != OK && ret != TM_ALREADY_STOPPED_APP) {
+			printf("ERROR: failed task_manager_stop in unregister_tasks_4 ERR: %d\n", ret);
+		}
+
 		ret = task_manager_unregister(task_info->handle, TM_RESPONSE_WAIT_INF);
-		TC_ASSERT_EQ("task_manager_unregister", ret, OK);
+		if (ret != OK) {
+			printf("ERROR: failed task_manager_unregister in unregister_tasks_4 ERR: %d\n", ret);
+		}
 
 		task_manager_clean_info(&task_info);
-		TC_ASSERT_NEQ("task_manager_clean_info", (tm_appinfo_t *)&task_info, NULL);
+		if ((tm_appinfo_t *)&task_info == NULL) {
```

Comment:
if task_manager_clean_info cleans the task_info, it will be NULL.

Reply:
In UTC it gave error when task_info is equated to NULL